far/string_utils.cpp

```diff
@@ -50,11 +50,16 @@ THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 //----------------------------------------------------------------------------
 
-string_view GetSpaces()
+string_view GetBlanks()
 {
 	return L" \t"sv;
 }
 
+string_view GetSpaces()
```

Comment:
Yeah! I removed `GetSpaces` for now. However, see my comment about the difference between `iswbalnk` and `iswspace`. One day we my want it back.

Please consider everything and make the final call, I will do whatever you say. My personal opinion is that we should consistently use `GetSpaces() + GetEols() + Global->Opt->strWordDiv.Get()` as word separators, but I very well may miss some subtle scenarios. I will change Viewer Search accordingly. Other places better to change in a separate PR. Please create and issue and assign to myself.

I added `IsSpaceOrEos` for the symmetry (and used it in `is_word_div`). Then I noticed this record in changelog 🤣

```
svs 26.05.2009 23:56:23 +0300 - build 948
. . .
2. Добавлена IsSpaceOrEos(), руками не мацать, нужна.
```

Reply:
Let's go with minimal changes that would allow the viewer search to work as expected.
Blanks vs spaces is not the goal of this PR after all. We can explore possible approaches later if needed.

Reply:
Sure, Removing `IsSpaceOrEos` in the next iteration.